sistr/1.1.2/README.md

```diff
@@ -0,0 +1,25 @@
+# SISTR container
+
+Main tool: [sistr](https://github.com/phac-nml/sistr_cmd)
+  
+Code repository: https://github.com/phac-nml/sistr_cmd
+
+Additional tools:
+- blast : 2.12.0+ 
+- mafft : v7.490
+- mash : 2.3
+
```

Comment:
could you please add pandas v1.3.5 and numpy 1.21.5? They seem to be important python packages